pkgs/os-specific/darwin/mysides/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, libarchive, p7zip }:
+
+stdenv.mkDerivation rec {
+  pname = "mysides";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/mosen/mysides/releases/download/v${version}/mysides-${version}.pkg";
```

Comment:
Would you consider trying to build from sources with `fetchFromGithub` and `xcbuildHook`?

Reply:
I haven't been able to get this built on my machine. Beside that I thought it's easier to install by all the people that have not installed the full XCode version, without that the build simply fails, right?

Reply:
xcbuild is a third-party open source implementation for xcodebuild